source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0038
+
+<b>Functional Description</b>
+
+The Header shall be included if the field &lt;TRANSIT OPERATION.MRN&gt; or &lt;TRANSIT OPERATION.LRN&gt; is PRESENT in the rejected message and shall be included in the CC917C.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0039
+
+<b>Functional Description</b>
+
+If the field &lt;TRANSIT OPERATION.LRN&gt; is PRESENT in the rejected message, this Data Item shall be filled in. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0040
+
+<b>Functional Description</b>
+
+If the field &lt;TRANSIT OPERATION.MRN&gt; is PRESENT in the rejected message, this Data Item shall be filled in. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0041
+
+<b>Functional Description</b>
+
+In a case when National risk is identified, by default, the value ‘X’ shall be included in &lt;RiskAnalysisIdentification.code&gt; when generated by the risk engine. However, if the sender would likely consider such information sent by the Office of Export /Office of Departure as useless for the recipient, then Office of Export /Office of Departure should use the value ‘N’.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0042
+
+<b>Functional Description</b>
+
+This D.G./D.I. is not used to report discrepancies. The Control Message always reports back D.G./D.I as at declaration message. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0045
+
+<b>Functional Description</b>
+
+The information in this Data Group/Data Item will override the information included in the CC015C (or in the latest CC013C, if any).
+
+<b>Technical Description</b>
+
+N/A
+
+##G0050
+
+<b>Functional Description</b>
+
+The Reference number shall include the ARC number or the fallback eAD reference number when the ‘Type’ of the ‘Additional reference’ is C651 or C658 respectively.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0057
+
+<b>Functional Description</b>
+
+Common Code List can be extended or restricted at national level. Purely national codes are not included in Common Domain messages.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0058
+
+<b>Functional Description</b>
+
+When /*/Consignment/HouseConsignment/ConsignmentItem/PreviousDocument/type is in SET {C651, C658} the Unique Body Reference (UBR) is required to be recorded in this field.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0061
+
+<b>Functional Description</b>
+
+The information presented in in this D.G. is related to Safety & Security and to the Binding Itinerary. In case of Binding itinerary, the information entered must include the list of codes of the countries between the Office of Departure and the Office of Destination. If more information is available about the countries visited by the means of transport since it's first place of loading until the last place of unloading, it should also be added for Safety & Security purpose only.
+
```

Comment:
Four issues:
- Repeated word `in`.
- Reword `In case of Binding itinerary` to `For Binding itineraries,` (note comma).
- Change `it's` to `its` (the former is a contraction of `it is` whereas the latter is a possessive).
- The phrase `purpose only` is problematic because it's not good style to put `only` at the end of a sentence. This might require some discussion but I reckon that the last clause of the last sentence should be reworded to `it should also be added but only for Safety & Security purposes`.